src/multiple_choice/template.py

```diff
@@ -361,8 +361,8 @@
 aio_model = "AllInOne (kprim, mc, sc)"
 aio_card = "AllInOne (kprim, mc, sc)"
 aio_fields = {
-    "title": "Title",
```

Comment:
I do think that's a legit improvement to be made. Can you please explain how changing the order making this field optional?

Reply:
Only the first field must not be empty (usually, it is a question): https://github.com/ankitects/anki/blob/main/qt/aqt/addcards.py#L171
You can find dupeOrEmpty() function return values explanation here: https://github.com/ankitects/anki/blob/3ba08d1189999c38f983288e980f39494c1c0057/pylib/anki/notes.py#L182

Reply:
Ah, perfect. Thanks for the reference!